src/invenio-records-js/controllers/InvenioRecordsCtrl.js

```diff
@@ -235,19 +240,13 @@ function InvenioRecordsCtrl($scope, $rootScope, $q, $window, $location,
       }
 
       // Trigger successful event for action
-      $rootScope.$broadcast('invenio.records.action.success', type);
+      $rootScope.$broadcast('invenio.records.action.success', _actions);
```

Comment:
Listeners of the `invenio.records.action.success` event should act based on the new "array-styled" format of actions (`['self', 'PUT']`).